doc/setup.md

```diff
@@ -16,21 +17,50 @@ When installing SkyPortal on Debian-based systems, 2 additional packages are req
 - libcurl4-gnutls-dev
 - libgnutls28-dev
 
-## Source download, Python environment
+## Package Managers
+Install `uv` and `Bun`:
+```
+# Install uv (faster Python package manager)
+curl -LsSf https://astral.sh/uv/install.sh | sh
+
+# Install bun (a javascript runtime, faster equivalent to `node+npm`)
+curl -fsSL https://bun.sh/install | bash
+```
+After installation, you'll need to add uv to your PATH.
+```angular2html
+source $HOME/.cargo/env # for bash, zsh, or sh shells
+source $HOME/.cargo/env.fish # for fish shell
+```
+For bun, reload your shell:
```

Comment:
many of our installation steps require that. I'd just tell people to reload their shell or open a new terminal and they can figure it out.